{{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml

```
@@ -44,7 +44,7 @@ jobs:
         fail-fast: false
         matrix:
           os: [macOS-latest, ubuntu-latest, windows-latest]
-          python-version: {{ '${{ fromJSON(needs.environment-config.outputs.python-matrix) }}' }}
+          python-version: {% raw %}${{ fromJSON(needs.environment-config.outputs.python-matrix) }}{% endraw %}
```

```suggestion
          python-version: {{ ${{ needs.environment-config.outputs.python-matrix }} }}
```
Try this for a sec? It might be that I've done a reasonable job storing it that it doesn't need reverting back.

---

Ah I tried that in the unstring commit I think, Jinja was unhappy about the $.